### DISCUSSION TITLE
Feature suggestion - Photo slide show as screen saver

### DISCUSSION THREAD

**Discussion by kvikindi** [KlipperScreen/KlipperScreen#324] [Ideas]
Hi all,

I've just had an utterly daft idea for displaying photos (from within a pre-set folder) as a screen saver.
There must be others out there that have their screens visible, and would enjoy having a rotating selection of daft images on it.

Just a thought.

**Comment by alfrix** [comment]
i think something like xscreensaver is what you are looking for, this functionality probably will never be in klipperscreen
